cfgov/apache/conf.d/redirects.conf

```diff
@@ -261,272 +216,17 @@ RedirectMatch permanent (?i)^/retirement/claiming-social-security/?$ /retirement
-
-# Ask CFPB redirects, June 2017
-# https://GHE/CFGOV/platform/issues/936
-Redirect permanent /askcfpb/361/  /askcfpb/2071/
-Redirect permanent /askcfpb/365/  /askcfpb/2071/
-Redirect permanent /askcfpb/367/  /askcfpb/2071/
-Redirect permanent /askcfpb/369/  /askcfpb/2071/
-Redirect permanent /askcfpb/371/  /askcfpb/2071/
-Redirect permanent /askcfpb/373/  /askcfpb/2071/
-Redirect permanent /askcfpb/375/  /askcfpb/2071/
-
-Redirect permanent /es/obtener-respuestas/c/comprar-una-casa/361/ /es/obtener-respuestas/c/comprar-una-casa/2071/que-es-el-acuerdo-hipotecario-nacional.html
-Redirect permanent /es/obtener-respuestas/c/comprar-una-casa/365/ /es/obtener-respuestas/c/comprar-una-casa/2071/que-es-el-acuerdo-hipotecario-nacional.html
-Redirect permanent /es/obtener-respuestas/c/comprar-una-casa/367/ /es/obtener-respuestas/c/comprar-una-casa/2071/que-es-el-acuerdo-hipotecario-nacional.html
-Redirect permanent /es/obtener-respuestas/c/comprar-una-casa/369/ /es/obtener-respuestas/c/comprar-una-casa/2071/que-es-el-acuerdo-hipotecario-nacional.html
-Redirect permanent /es/obtener-respuestas/c/comprar-una-casa/371/ /es/obtener-respuestas/c/comprar-una-casa/2071/que-es-el-acuerdo-hipotecario-nacional.html
-Redirect permanent /es/obtener-respuestas/c/comprar-una-casa/373/ /es/obtener-respuestas/c/comprar-una-casa/2071/que-es-el-acuerdo-hipotecario-nacional.html
-Redirect permanent /es/obtener-respuestas/c/comprar-una-casa/375/ /es/obtener-respuestas/c/comprar-una-casa/2071/que-es-el-acuerdo-hipotecario-nacional.html
-
 # Owning a Home redirects
 RewriteRule ^owningahome$ /owning-a-home$1 [L,R=301]
-Redirect permanent /owningahome /owning-a-home
-Redirect permanent /OWNING-A-HOME /owning-a-home
-Redirect permanent /OWNINGAHOME /owning-a-home
 
-Redirect permanent /mortgage-estimate /owning-a-home/mortgage-estimate/?utm_source=mortgageestimate&utm_medium=redirect&utm_campaign=OAHredirects
-Redirect permanent /mortgage-closing /owning-a-home/mortgage-closing/?utm_source=mortgageclosing&utm_medium=redirect&utm_campaign=OAHredirects
-Redirect permanent /owning-a-home/check-rates /owning-a-home/explore-rates/?utm_source=checkrates&utm_medium=redirect&utm_campaign=OAHredirects
 RedirectMatch permanent ^/owning-a-home/monthly-payment-worksheet[/]*$ /owning-a-home/resources/monthly_payment_worksheet.pdf
 
-## Redirecting old files stored directly on the server to standard document library locations
-Redirect permanent /owning-a-home/resources/adjustable_rate_loan_estimate.pdf /f/documents/cfpb_buying-a-house_adjustable-rate-loan-estimate.pdf
-Redirect permanent /owning-a-home/resources/checklist_mortgage_closing.pdf /f/documents/cfpb_buying-a-house_mortgage-closing_checklist.pdf
-Redirect permanent /owning-a-home/resources/monthly_payment_worksheet.pdf /f/documents/cfpb_buying-a-house_monthly-payment_worksheet.pdf
-Redirect permanent /owning-a-home/resources/mortgage_closing_forms.pdf /f/documents/cfpb_buying-a-house_closing-forms_guide.pdf
```

Comment:
There might be some redirect chains going on here, or it might have been something weird in my local environment, but none of these `/owning-a-home/resources/` redirects were working for me. They all ended up redirecting to `/static/owning-a-home/resources/[whatever]`. Not sure if that should block this PR or not since it's hard to tell what's environment issues and what's actual redirect issues.

Reply:
Super weird. As far as I can tell the Wagtail-built redirects are set up properly (i.e. the "to" field URL resolves as it should). I can always cross-check these after the PR merges.